config/client.go

```diff
@@ -149,6 +151,23 @@ type CoprocessorCache struct {
 	AdmissionMinProcessMs uint64 `toml:"admission-min-process-ms" json:"-"`
 }
 
+// CircuitBreakerSettings is the config for default circuit breaker settings excluding the error rate
+type CircuitBreakerSettings struct {
```

Comment:
How about referring to `config/retry/backoff.go`? We can use NewCircuitBreakerWithVars or something like that and only add a var for the error rate threshold. The other configs can be hard coded and we don't need to support changing them through the config file. @Tema @MyonKeminta @okJiang WDYT?

Reply:
How about propagating the error rate instead of the whole `Settings`? As for the other configs, as we discussed before, we can make them hard-coded temporarily. If we do need to change them, then we can make them configurable. Right now, it's better to hide them to reduce the complexity.

Reply:
LGTM, /cc @okJiang

Reply:
LGTM